components/hydrator/internal/subject/cert_subject.go

```diff
@@ -140,17 +139,7 @@ func ExternalCertIssuerSubjectMatcher(externalSubjectConsts ExternalIssuerSubjec
 		if cert.GetCountry(subject) != externalSubjectConsts.Country || cert.GetOrganization(subject) != externalSubjectConsts.Organization {
 			return false
 		}
-		orgUnitRegex := regexp.MustCompile(externalSubjectConsts.OrganizationalUnitPattern)
```

Comment:
How was this working before when the `GetPossibleRegexTopLevelMatches` did not exist?

Reply:
Old reference in go mod from a commit part of the previous PR history.